documentation/docs/pages/threat-model-viz.css

```
@@ -0,0 +1,436 @@
+  --node-ipr: #b8860b;
+  --node-sp: #d6691f;
+  --node-destination: #d63b3b;
+  --node-vpn: #607a76;
+  --node-internet: #7c8a86;
+
+  --mode-direct: #d63b3b;
+  --mode-vpn: #607a76;
+  --mode-dvpn: #1f7fd6;
+  --mode-mixnet: #8a3bd6;
+
+  --font-mono: ui-monospace, "SF Mono", "JetBrains Mono", "Fira Code", "Cascadia Code", Menlo, Consolas, monospace;
```

Stylelint (17.14.1)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)